apisix/admin/plugins.lua

```diff
@@ -42,7 +44,14 @@ end
 
 function _M.get(name)
     local arg = get_uri_args()
-    if arg and arg["all"] == "true" then
```

Comment:
This method should be kept. Let's add WARNING log about it will be deprecated soon

Reply:
I added the warning log @monkeyDluffy6017